src/pages/home/report/ContextMenu/PopoverReportActionContextMenu.js

```diff
@@ -203,7 +202,6 @@ class PopoverReportActionContextMenu extends React.Component {
         this.setState({
             reportID: 0,
             reportAction: {},
-            selection: '',
```

Comment:
Didn't get this change?

Reply:
This is an object now, which we expect to be null if it bears no value.

Reply:
So if you don't reset it, it will take old value.

Reply:
The selection signature has changed, so all old calls were converted to {text: original-value}

Reply:
But how the reset is handled for selection now?